src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/CountReadsSpark.java

```diff
@@ -36,7 +36,6 @@ protected void runTool(final JavaSparkContext ctx) {
         final JavaRDD<GATKRead> reads = getReads();
 
         final long count = reads.count();
-        System.out.println(count);
```

Comment:
GATK users are accustomed to having the `Count*` tools print the count at the end of their run, rather than being forced to create an output file.